src/main/java/uk/nhs/digital/nhsconnect/nhais/mesh/MongoScheduler.java

```diff
@@ -0,0 +1,67 @@
+package uk.nhs.digital.nhsconnect.nhais.mesh;
+
+import static org.springframework.data.mongodb.core.query.Criteria.where;
+import static org.springframework.data.mongodb.core.query.Query.query;
+
+import java.time.LocalDateTime;
+
+import org.bson.Document;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoOperations;
+import org.springframework.data.mongodb.core.query.Update;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import com.mongodb.client.result.UpdateResult;
+
+@Component
+public class MongoScheduler {
+
+    @Autowired
+    private MongoOperations mongoOperations;
+
+    @Autowired
+    private MeshClient meshClient;
+
+    private static final String MESH_TIMESTAMP = "mesh_timestamp";
+    private static final String TIMESTAMP = "timestamp";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MongoScheduler.class);
+
+    @Scheduled(fixedRate = 60000)
```

Comment:
Why not https://github.com/lukas-krecan/ShedLock ?

Reply:
If current implementation works, then we could use ShedLock later (in another ticket)

Reply:
Agreed to do it in another PR